features/fixtures/policy_and_engagement.json

```diff
@@ -0,0 +1,114 @@
+{
+  "analytics_identifier":null,
+  "base_path":"/policy-and-engagement",
+  "content_id":"622e9691-4b4f-4e9c-bce1-098b0c4f5ee2",
+  "content_purpose_document_supertype":"navigation",
+  "document_type":"finder",
+  "email_document_supertype":"other",
+  "first_published_at":"2018-11-07T14:47:48.000+00:00",
+  "government_document_supertype":"other",
+  "locale":"en",
+  "navigation_document_supertype":"other",
+  "phase":"alpha",
+  "public_updated_at":"2018-11-07T14:47:48.000+00:00",
+  "publishing_app":"finder-frontend",
+  "rendering_app":"finder-frontend",
+  "schema_name":"finder",
+  "search_user_need_document_supertype":"government",
+  "title":"Policy papers and consultations",
+  "updated_at":"2018-11-07T14:47:48.000+00:00",
+  "user_journey_document_supertype":"finding",
+  "withdrawn_notice":{
+
+  },
+  "publishing_request_id":"",
+  "links":{},
+  "description":"Find policy papers and consultations from government",
+  "details":{
+    "document_noun":"result",
+    "filter":{
+    },
+    "format_name":"document",
+    "show_summaries":true,
+    "sort": [
+      {
+        "name": "Most viewed",
+        "key": "-popularity"
+      },
+      {
+        "name": "Relevance",
+        "key": "-relevance"
+      },
+      {
+        "name": "Updated (newest)",
+        "key": "-public_timestamp",
+        "default": true
+      },
+      {
+        "name": "Updated (oldest)",
+        "key": "public_timestamp"
+      }
+    ],
+    "facets":[
+      {
+        "filter_key": "all_part_of_taxonomy_tree",
+        "keys": ["level_one_taxon", "level_two_taxon"],
+        "name": "topic",
+        "short_name": "topic",
+        "type": "taxon",
+        "display_as_result_metadata": false,
+        "filterable": true,
+        "preposition": "about"
+      },
+      {
+        "key": "organisations",
+        "name": "Organisation",
+        "short_name": "From",
+        "preposition": "from",
+        "type": "text",
+        "display_as_result_metadata": true,
+        "filterable": true
+      },
+      {
+        "key": "world_locations",
+        "name": "World location",
+        "preposition": "in",
+        "type": "text",
+        "display_as_result_metadata": true,
+        "filterable": true
+      },
+      {
+        "key": "content_store_document_type",
+        "name": "Consultation status",
+        "preposition": "with consultation status",
+        "type":"dropdown_select",
+        "display_as_result_metadata": true,
+        "filterable": true,
+        "allowed_values": [
+          {
+            "text": "All consultations",
+            "value": "[\"open_consultation\",\"closed_consultation\",\"consultation_outcome\",\"impact_assessment\",\"case_study\",\"policy_paper\"]",
```

Comment:
Our rules now differ from `govuk_document_types`. We're labelling an impact assessment, policy paper and case study as a consultation. I'm not sure that is correct?

See https://github.com/alphagov/govuk_document_types/blob/master/data/supertypes.yml#L458-L467

```yaml
    - id: policy
      document_types:
        - impact_assessment
        - case_study
        - policy_paper
    - id: consultations
      document_types:
        - open_consultation
        - closed_consultation
        - consultation_outcome
```

Reply:
I followed the supergroup for `policy_and_engagement` which includes these document types:

```
    - id: policy_and_engagement
      document_types:
        - impact_assessment
        - case_study
        - policy_paper
        - open_consultation
        - closed_consultation
        - consultation_outcome
```

https://github.com/alphagov/govuk_document_types/blob/master/data/supertypes.yml#L559-L566

The default label for the consultations facet is 'all consultations', which I intended to imply 'everything tagged to the policy and engagement supergroup'. Would it be worth us asking a content specialist about this?

Reply:
I need a follow up PR to hide the facet tag when the default option is selected for this facet, currently this only happens if the value is `""`.

Reply:
Here's that Pr which kind of solves the problem: https://github.com/alphagov/finder-frontend/pull/809.

So we can remove "All consultations" as being a group of all document_types included in the supergroup and have something similar to the research and stats finder.